retriever/engines/postgres.py

```diff
@@ -108,20 +108,20 @@ def insert_data_from_file(self, filename):
         p_bulk = [self.check_bulk_insert(), self.table.delimiter, self.table.header_rows]
         if p_bulk == [True, ",", 1]:
             columns = self.table.get_insert_columns()
-            filename = os.path.abspath(filename)
+            filename = os.path.normpath(os.path.abspath(filename))
             statement = """
 COPY """ + self.table_name() + " (" + columns + """)
-FROM '""" + filename.replace("\\", "\\\\") + """'
+FROM '""" + filename + """'
 WITH DELIMITER ','
 CSV HEADER;"""
             try:
                 self.execute("BEGIN")
                 self.execute(statement)
                 self.execute("COMMIT")
+                print("Bulk insert on .. ", self.table_name())
                 return True
-            except BaseException:
+            except Exception as e:
```

Comment:
@henrykironde I don't think we need to assign the exception to a variable if we are not printing it.